### PR TITLE
check CE fields emitted correctly

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Check ce1Complete is set, as was coming through as null sometimes

# What has changed
Now tests for ce1Complete

# How to test?
Run with master except the new fieldworkAdapter branch: https://github.com/ONSdigital/census-rm-fieldwork-adapter/pull/25

# Links
https://trello.com/c/gIGqWwlV/493-fixes-to-ce

### CODE REVIEW

File: acceptance_tests/features/field_reminder.feature [hunk 1]
```diff
@@ -10,4 +10,4 @@ Feature: Reminder messages are emitted to Field Work Management Tool
   Scenario: send community estab cases to field
     Given sample file "sample_for_ce_stories.csv" is loaded successfully
     When an action rule for community estabs is set 5 seconds in the future
-    Then the action instruction messages are emitted to FWMT where the case has a "caseType" of "CE"
+    Then action instruction emitted to FWMT where case has a "caseType" of "CE" and CEComplete is "false"
```
Comment (AdamHawtin) [inline acceptance_tests/features/field_reminder.feature]: Don't think you should remove the 'the', scenarios are supposed to be full sentences.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Just a grammar comment

File: acceptance_tests/features/field_reminder.feature (hunk 1 above)
Comment (LukeLoze1) [reply acceptance_tests/features/field_reminder.feature]: Fair enough, I removed it when I was trying to shorten a longer version (>80), doesn't apply any more

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
I'm getting a failure in an unrelated test due to the test code changes:
```python
  Scenario: Generate individual cases and check that no actions rules are triggered for them                                          # acceptance_tests/features/fulfilment_request.feature:84
    Given sample file "sample_individual_case_spec.csv" is loaded successfully                                                        # acceptance_tests/features/steps/sample.py:26 0.087s
    And an action rule of type "FIELD" is set 5 seconds in the future                                                                 # acceptance_tests/features/steps/action_rules.py:9 0.092s
    When a UAC fulfilment request "UACIT1" message for a created case is sent                                                         # acceptance_tests/features/steps/fulfilment_request.py:67 0.028s
    Then the action instruction messages for only the HH case are emitted to FWMT where the case has a "treatmentCode" of "HH_QF2R1E" # acceptance_tests/features/steps/field_reminder.py:11 0.000s
      Traceback (most recent call last):
        File "/Users/adamhawtin/.local/share/virtualenvs/census-rm-acceptance-tests-k93M4obU/lib/python3.6/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/Users/adamhawtin/.local/share/virtualenvs/census-rm-acceptance-tests-k93M4obU/lib/python3.6/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
      TypeError: fwmt_messages_received() got an unexpected keyword argument 'treatment_code'
```

File: acceptance_tests/features/field_reminder.feature (hunk 1 above)
Comment (LeoKHoward) [reply acceptance_tests/features/field_reminder.feature]: My only observation was grammatical too...

Would it make more sense to say "Then the action instruction IS emitted to FWMT..."

**Review by LeoKHoward — APPROVED** [review]

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**